external-events/pnp/pnp.public.output.change-notification-events.v1.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$id": "pnp.public.output.change-notification-events.v1", 
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$ref" : "#/definitions/record:hiiretail.contracts.changenotifications.ChangeNotificationEvent",
```

Comment:
I dont think the $ref will work in this case. May be we will delete it?

Reply:
Well, it will. First of all, it is used this way by the Java library that we use for schema generation. Second of all, I have used this [online generator](https://json-schema-faker.js.org/) to check whether it can generate a json based on such schema and it could. Though this way of using $ref keyword may be an outdated one since the library is not maintained for 10 years already, but if this works then it does not matter that much.
![image](https://github.com/user-attachments/assets/aaab9b0f-5688-43b1-8fbb-d55417499d00)